### ISSUE TITLE
Tabby freezes when trying to create or open an existing profile

### ISSUE THREAD

**Issue by sheshadriv32** [Eugeny/tabby#8605]
Describe: Tabby freezes when trying to create or open existing profile.
You have force close the terminal or terminate it with task manager.

Version: 1.0.197
OS: Windows 10 Enterprise Version 21H2
Reproducibility: Very easy

Steps to reproduce:
1. Open any session with tabby.
2. Try to create/open a new session using Window button on the toolbar or through settings menu.
3. Tabby won't respond. You have to press escape button to come out and in worse case force close it or terminate via task manager.

**Comment by ssepssep** [comment]
If you relaxed theme? I changed the theme to standard and it worked out.

**Comment by SheshadriVenkatesh** [comment]
No, I was using the snazzy theme. Switched to Standard legacy theme which seems to have resolved the issue.
